tests/integration/models/device.spec.coffee

```diff
@@ -500,28 +510,30 @@ describe 'Device Model', ->
 
 			it 'should be able to set the location of a device by id', ->
 				balena.models.device.setCustomLocation @device.id,
-					latitude: 41.383333
-					longitude: 2.183333
+					latitude: 42.383333
```

Comment:
I am curious why you changes the long/lat values :)

Reply:
Setcases >> Barcelona!

Reply:
Previously, all method variants were setting the same value and as a result we had to re-create the device, in order to test that the value was properly set by all test cases.
Now, by setting different values by each method variant, we can test that each call was successful w/o the need of re-creating the device.

Reply:
Also, I didn't check the original coordinates, but Barcelona makes sense, since Tim was there :)